app/src/main/res/layout/audio_fragment.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <data>
+
+    <variable
+      name="languageCode"
```

Comment:
Does LanguageInterface have a languageCode field? It seems to have a method for retrieving the current language code.